pkg/util/docker/dockerfile/dockerfile.go

```diff
@@ -0,0 +1,55 @@
+package dockerfile
```

Comment:
i don't mind it being under util, just wondering why it's not in util/docker?  (which i assumed was an existing package, i see it's not..but i think it would make sense to create that package and put this there, rather than getting so specific)